Realm/RLMObjectSchema.mm

```diff
@@ -268,6 +312,7 @@ - (id)copyWithZone:(NSZone *)zone {
 
     // call property setter to reset map and primary key
     schema.properties = [[NSArray allocWithZone:zone] initWithArray:_properties copyItems:YES];
+    schema.computedProperties = [[NSArray allocWithZone:zone] initWithArray:_computedProperties copyItems:YES];
```

Comment:
Does this need to deep copy? IIRC for persisted properties that's purely for the `column` property, and that doesn't appear to be used for computed ones.